week-1/Homework/extra/1-currency-conversion.js

```diff
@@ -5,7 +5,9 @@
   Write a function that converts a price to USD (exchange rate is 1.4 $ to £)
 */
 
-function convertToUSD() {}
+function convertToUSD(tax) {
+  return tax  20% + sales;
+}
```

Comment:
Actually the instruction is telling to convert the price to USD, Multiplying the amount or the Price by 1.4. Could you please fix that and do not forget the reset of the exercise.